serverless/cmd/feeder/github/main.go

```diff
@@ -201,6 +203,18 @@ func feedOnce(ctx context.Context, opts *options, forkRepo *git.Repository, ghCl
 	return createPR(ctx, opts, ghCli, fmt.Sprintf("Witness @ %d", wCp.Size), opts.feederRepo.owner+":"+branchName, "master")
 }
 
+// cpID returns a stable identifier for a given checkpoint and set of known signatures.
```

Comment:
Maybe nit, maybe important: this says a "set of", but sets are unordered and there is no sorting of the sigs here, hence the identifier generated doesn't look like it is stable if the sigs are in different orders.

Options::
 * s/set/list
 * sort the sigs
 * add docs here if `n.Sigs` is already sorted in some stable way

Reply:
Should be stably sorted, added a comment to that effect.